magento/pwa-studio
# Peregrine implementation/extract view state

## Description

This PR moves state relevant to views out of the store and into the components that care about it. See the issue for more information.

## Related Issue





Closes #1600.

## Verification Steps

This touches a _lot_ of code. A regression test should cover our bases on top of unit tests, but I encourage you to try various actions/ buttons within the app that utilize redux to ensure I didn't break anything.

## Screenshots / Screen Captures (if appropriate)

## Checklist

* I have updated the documentation accordingly, if necessary.
* I have added tests to cover my changes, if necessary.
